ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/WriteService.scala

```diff
@@ -95,13 +96,17 @@ trait WriteService
     * @param estimatedInterpretationCost Estimated cost of interpretation that may be used for
     *                                    handling submitted transactions differently.
     * @param telemetryContext            Implicit context for tracing.
+    * @param loggingContext            Implicit context for logging.
```

Comment:
Nit: format

Reply:
I think I would be OK with just deleting the Scaladoc for the implicits. They don't tell us anything.